_posts/en/newsletters/2021-07-07-newsletter.md

```diff
@@ -118,7 +118,12 @@ BOLTs][bolts repo].*
   extend descriptor ranges, reactivate inactive descriptors, and make other
   updates using the `importdescriptors` wallet RPC.
 
-- [C-Lightning #4610][] opts: Add option to register extra TLV types to accept FIXME:dongcarl
+- [C-Lightning #4610][] adds an `--experimental-accept-extra-tlv-types`
+  command-line option which allows users to specify a list of even TLV types
+  that `lightningd` should pass through for plugins to handle. Previously,
+  `lightningd` considered all evenly-typed TLV messages to be invalid. This
+  change allows for plugins to define and handle their own custom TLV types
+  unknown to `lightningd`.
```

Comment:
Not 100% sure if the rationale described in this last sentence is accurate. Can anyone sanity-check?

Reply:
LGTM.  I added two words elsewhere in the description for clarity.